src/object_classes_permissions.md

```diff
@@ -2143,6 +2150,26 @@ Manage the Binder IPC service.
 - Transfer a binder reference to another process (can A transfer a binder
   reference to B?).
 
+### *io_uring*
+
+Manage security-sensitive usages of the *io_uring* subsystem.
+
+**Permission** - 3 unique permissions:
+
+*override_creds*
+
+- Use another process's credentials via an *io_uring* personality (Can A use B's
+  credentials when calling ***io_uring_enter**(2)*?).
```

Comment:
The `override_creds` permission applies not just when `io_uring_enter(2)` is called, but anytime an io_uring operation is submitted and the submitter's credentials do not match the credentials requested by the io_uring personality.

As a reminder, with io_uring it is possible to submit new operations without calling `io_uring_enter(2)`.